src/scm/github.js

```diff
@@ -0,0 +1,427 @@
+'use strict';
+
+const Github = class Github {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/leonhartX/gas-github/blob/9b7bf2b597e67365f1e9002c784237b58657e4cc/src/scm/github.js#L3"></span>‘Github’ is assigned a value but never used.</td>
    </tr>
  </tbody>
</table>